snappy/app/netfilter_test.go

```diff
@@ -92,24 +92,37 @@ func (s *FilterSuite) TestFilterHandleRequest(c *C) {
 	req.RemoteAddr = "127.0.0.1:80"
 
 	http.DefaultServeMux.ServeHTTP(rec, req)
-	c.Assert(rec.Code, Equals, http.StatusForbidden)
+	// device connected networks are always authorized.
```

Comment:
Please make sure your use of capitalization at the beginning of each sentence is consistent and consistent use of punctuation at the end.